### ISSUE TITLE
Difference in output dimensions

### ISSUE THREAD

**Issue by netw0rkf10w** [amir-abdi/keras_to_tensorflow#42]
Hello,

First of all thank you very much for the valuable code!

I used your code to convert my Keras model but there is some inconsistency in the output dimensions between the models.

Checking for the original model

```
model = load_model('keras_model.h5', compile=False)
print(model.output)
```
I obtained:

> [<tf.Tensor 'conv2d_10/BiasAdd:0' shape=(?, ?, ?, 33) dtype=float32>, <tf.Tensor 'conv2d_13/BiasAdd:0' shape=(?, ?, ?, 33) dtype=float32>]

So the output of the model is a list of 2 tensors.

Now for the converted model in Tensorflow:
```
graph = tf.Graph()
with graph.as_default():
    graph_def = tf.GraphDef()
    with tf.gfile.GFile('tf_model.pb', 'rb') as fid:
        graph_def.ParseFromString(fid.read())
        tf.import_graph_def(graph_def, name='')

        # Print the name of operations in the session
        for op in graph.get_operations():
            print("Operation Name:",op.name)
            print("Tensor Stats:",str(op.values()))
```
The last few output lines of the above code is:

> Operation Name : leaky_re_lu_9/LeakyRelu/mul
> Tensor Stats : (<tf.Tensor 'leaky_re_lu_9/LeakyRelu/mul:0' shape=(?, ?, ?, 512) dtype=float32>,)
> Operation Name : leaky_re_lu_9/LeakyRelu/Maximum
> Tensor Stats : (<tf.Tensor 'leaky_re_lu_9/LeakyRelu/Maximum:0' shape=(?, ?, ?, 512) dtype=float32>,)
> Operation Name : conv2d_10/convolution
> Tensor Stats : (<tf.Tensor 'conv2d_10/convolution:0' shape=(?, ?, ?, 33) dtype=float32>,)
> Operation Name : conv2d_10/BiasAdd
> Tensor Stats : (<tf.Tensor 'conv2d_10/BiasAdd:0' shape=(?, ?, ?, 33) dtype=float32>,)
> Operation Name : output_node0
> Tensor Stats : (<tf.Tensor 'output_node0:0' shape=(?, ?, ?, 33) dtype=float32>,)

The last line is the output, it is a single tensor.

In conclusion, the Keras output is a list of 2 tensors, while the Tensorflow output is a single tensor.

Could you please help?
Thank you very much in advance!

**Comment by netw0rkf10w** [comment]
I forgot to set the number of outputs to 2 when running the script, sorry. Problem solved.
